README.md

```diff
@@ -2,7 +2,7 @@
 
 [![Build Status](https://travis-ci.org/esl/MongoosePush.svg?branch=master)](https://travis-ci.org/esl/MongoosePush) [![Coverage Status](https://coveralls.io/repos/github/esl/MongoosePush/badge.svg?branch=master)](https://coveralls.io/github/esl/MongoosePush?branch=master) [![Ebert](https://ebertapp.io/github/esl/MongoosePush.svg)](https://ebertapp.io/github/esl/MongoosePush)
 
-**MongoosePush** is simple (seriously) **REST** service written in **Elixir** providing ability to **send push
+**MongoosePush** is simple, **RESTful** service written in **Elixir**, providing ability to **send push
```

Comment:
**MongoosePush** is a simple, **RESTful** service written in **Elixir**, providing ability to **send push notifications** to `FCM` (Firebase Cloud Messaging) and/or `APNS` (Apple Push Notification Service) via their `HTTP/2` API.